app/src/main/java/com/springsamurais/toyswap/service/APIService.kt

```diff
@@ -14,7 +14,7 @@ interface APIService {
     fun getListings(): Call<MutableList<Listing?>?>?
 
     @Multipart
-    @POST("listings")
+    @POST("/api/listings")
```

Comment:
Does this work? The retrofit should hit the main endpoint: `http://10.0.2.2:8080/api/v1/` then the annotated endpoints should just be a single extension to that endpoint: `listings`